src/app/modules/twilio/twilio.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { StatusCodes } from 'http-status-codes'
+import Twilio from 'twilio'
+
+import { ITwilioSmsWebhookBody } from 'src/types/twilio'
+
+import { ControllerHandler } from '../core/core.types'
+
+import { logFailedSmsDelivery } from './twilio.service'
+
+const smsDeliveryFailedStatus = ['undelivered', 'failed']
```

Comment:
I think we can just inline this; are there any stronger types too? We should also link to the documentation.

Reply:
Currently, it seem like its provided only as string values from the example seen here: https://www.twilio.com/docs/usage/webhooks/sms-webhooks. 

Checked the twilio typescript support, dosen't seem to have stronger types for it either